src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -225,6 +243,43 @@ public function load(array $configs, ContainerBuilder $container)
             'Symfony\\Bundle\\FrameworkBundle\\Controller\\Controller',
         ));
 
+        $container->registerForAutoconfiguration(Command::class)
+            ->addTag('console.command');
+        $container->registerForAutoconfiguration(ResourceCheckerInterface::class)
+            ->addTag('config_cache.resource_checker');
+        $container->registerForAutoconfiguration(ServiceSubscriberInterface::class)
+            ->addTag('container.service_subscriber');
+        $container->registerForAutoconfiguration(AbstractController::class)
+            ->addTag('controller.service_arguments');
+        $container->registerForAutoconfiguration(DataCollectorInterface::class)
+            ->addTag('data_collector');
+        $container->registerForAutoconfiguration(FormTypeInterface::class)
```

Comment:
I don't think it makes any difference, but I can see the appeal of organizing these in that way (i.e. adding the serializer-related stuff inside `registerSerializerConfiguration`). We could always move it later without a BC break since adding these when the form is disabled (for example) has no effect.